config/locales/spotlight.en.yml

```diff
@@ -780,7 +780,8 @@ en:
       import:
         description: You can import and export Rails-based YAML files containing the translated label values for the currently selected language for use in internationalization tools such as Transifex and i18n-tools. This feature does not include importing or exporting translations for Pages.
         export_label: Export a YAML file
-        export_submit: Export translations
+        export_current_locale: Export %{language}
+        export_default_locale: Export %{language}
```

Comment:
The values are equivalent here, but I created separate keys in case someone wants to override these downstream.